src/main/resources/db/changelog/alert/5.0.0/changelog-configuration-initialization.xml

```diff
@@ -22,4 +22,16 @@
             CALL DEFINE_FIELD('jira.access.token', TRUE, 'channel_jira', 'GLOBAL');
         </sql>
     </changeSet>
+    <!-- Register Distribution Jira Cloud Fields. -->
+    <changeSet author="psantos" id="1558524516692-3">
+        <sql dbms="h2" stripComments="true">
+            CALL DEFINE_FIELD('jira.add.comments', FALSE, 'channel_jira', 'DISTRIBUTION');
```

Comment:
For these distribution specific fields do we want channel. as the prefix?  i.e. channel.jira.add.comments
For the HipChat and Slack distribution fields we do that.  So to be consistent we can do that here as well. We also have the FIELD_CONTEXTS table that creates the mapping between the field ID and the Context.

Reply:
I think we should add the prefix for consistency.

Reply:
I would like this as we could have a Jira provider in the future and the prefix would help avoid conflicts.

Reply:
ok no problem.  I prefer the consistency as well but wanted to make sure I wasn't being too particular.